Detectors/CPV/simulation/src/Detector.cxx

```diff
@@ -183,43 +168,43 @@ Bool_t Detector::ProcessHits(FairVolume* v)
   float pX = pd[0];
   float pZ = pd[2];
   float pNorm = -pd[1];
-  float dZY = pZ / pNorm * kCPVGasThickness;
-  float dXY = pX / pNorm * kCPVGasThickness;
+  float dZY = pZ / pNorm * o2::cpv::CPVSimParams::Instance().mCPVGasThickness;
+  float dXY = pX / pNorm * o2::cpv::CPVSimParams::Instance().mCPVGasThickness;
   float rnor1 = 0., rnor2 = 0.;
   gRandom->Rannor(rnor1, rnor2);
-  float eloss = kdEdx * (1 + kDetR * rnor1) *
-                TMath::Sqrt((1. + (pow(dZY, 2) + pow(dXY, 2)) / pow(kCPVGasThickness, 2)));
-  float zhit1 = hitZ + kCPVActiveSizeZ / 2 - dZY / 2;
-  float xhit1 = hitX + kCPVActiveSizeX / 2 - dXY / 2;
+  float eloss = o2::cpv::CPVSimParams::Instance().mdEdx * (1 + o2::cpv::CPVSimParams::Instance().mDetR * rnor1) *
+                TMath::Sqrt((1. + (pow(dZY, 2) + pow(dXY, 2)) / pow(o2::cpv::CPVSimParams::Instance().mCPVGasThickness, 2)));
+  float zhit1 = hitZ + o2::cpv::CPVSimParams::Instance().mCPVActiveSizeZ / 2 - dZY / 2;
+  float xhit1 = hitX + o2::cpv::CPVSimParams::Instance().mCPVActiveSizeX / 2 - dXY / 2;
   float zhit2 = zhit1 + dZY;
   float xhit2 = xhit1 + dXY;
 
-  int iwht1 = (int)(xhit1 / kCelWr); // wire (x) coordinate "in"
-  int iwht2 = (int)(xhit2 / kCelWr); // wire (x) coordinate "out"
+  int iwht1 = (int)(xhit1 / o2::cpv::CPVSimParams::Instance().mCellWr); // wire (x) coordinate "in"
+  int iwht2 = (int)(xhit2 / o2::cpv::CPVSimParams::Instance().mCellWr); // wire (x) coordinate "out"
```

Comment:
Here, in for better readability consider saying `auto& cpvparam = o2::cpv::CPVSimParams::Instance();` once on top and just use `cpvparam.mdEdx` etc. in all other placed.